scripts/v120_dynamic.py

```diff
@@ -0,0 +1,15 @@
+from compas.geometry import Point
+from compas_view2 import app
+from compas_view2.app.timer import Timer
+
+
+def movepoint():
+    print('test')
+    obj._data[0] += 1
+    viewer.view.update()
+
+
+viewer = app.App()
+obj = viewer.add(Point(0, 0, 0))
+viewer.timer = Timer(interval=1000, callback=movepoint)
```

Comment:
would be better to do this with a decorator directly on the callback function

Reply:
yep i agree, how about something like: 
```python
@animate(interval=1000)
function callback(frame): # maybe also pass in the frame count here 
    print("This is frame no.", frame)
    obj._data[0] += 1
    # Should we have update functions be called automatically in background?
    # obj.update()
    # viewer.view.update()
    
```